.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['1.3', '1']

```

Comment:
Would it make sense to test on more recent versions as well? At least 1.6 should be there I guess

Reply:
“1” actually always represents the latest stable. Currently, that would be 1.6.1.